### PR TITLE
[MINOR][DOCS] Fix some typos

### PR DESCRIPTION
**Pull request by plusplusjiajia**
### What changes were proposed in this pull request?

This PR proposes a few typos:
1. Sparks => Spark's
2. parallize => parallelize
3. doesnt => doesn't

### CODE REVIEW

**Comment by wangyum** [comment]
ok to test

File: sql/core/src/main/scala/org/apache/spark/sql/execution/ui/SQLAppStatusStore.scala [hunk 1]
```diff
@@ -133,7 +133,7 @@ class SparkPlanGraphNodeWrapper(
     val cluster: SparkPlanGraphClusterWrapper) {
 
   def toSparkPlanGraphNode(): SparkPlanGraphNode = {
-    assert(node == null ^ cluster == null, "One and only of of nore or cluster must be set.")
+    assert(node == null ^ cluster == null, "One and only one of node or cluster must be set.")
```
Comment (srowen) [inline sql/core/src/main/scala/org/apache/spark/sql/execution/ui/SQLAppStatusStore.scala]: You could say "Exactly one of ..." while here

**Review by srowen — APPROVED** [review]
This is fine; any other examples you can find?

File: sql/core/src/main/scala/org/apache/spark/sql/execution/ui/SQLAppStatusStore.scala (hunk 1 above)
Comment (plusplusjiajia) [reply sql/core/src/main/scala/org/apache/spark/sql/execution/ui/SQLAppStatusStore.scala]: > You could say "Exactly one of ..." while here

Thanks, done.

**Comment by wangyum** [comment]
retest this please

**Review by srowen — APPROVED** [review]

**Comment by srowen** [comment]
Merged to master

**Pull request closed without merging**